class-wc-gateway-komoju.php

```diff
@@ -14,186 +14,195 @@
-		$this->web_money			= $this->get_option( 'web_money' );
-		$this->konbini				= $this->get_option( 'konbini' );
-		$this->bank_transfer		= $this->get_option( 'bank_transfer' );
-		$this->pay_easy				= $this->get_option( 'pay_easy' );
-		self::$log_enabled    		= $this->debug;
-		// Load the settings.
-		$this->init_form_fields();
-		$this->init_settings();
-		// Define user set variables
-		$this->title        = $this->get_option( 'title' );
-		$this->description  = $this->get_option( 'description' );
-		$this->instructions = $this->get_option( 'instructions', $this->description );
-		// Filters
-		// Actions
-		add_action( 'woocommerce_update_options_payment_gateways_' . $this->id, array( $this, 'process_admin_options' ) );
-		if ( ! $this->is_valid_for_use() ) {
-			$this->enabled = 'no';
-			WC_Gateway_Komoju::log( 'is not valid for use. No IPN set.' );
-		} else {
-			include_once( 'includes/class-wc-gateway-komoju-ipn-handler.php' );
-			new WC_Gateway_Komoju_IPN_Handler( $this->webhookSecretToken, $this->secretKey, $this->invoice_prefix );
-		}
-
-	}
-
-	/**
-	 * Logging method
-	 * @param  string $message
-	 */
-	public static function log( $message ) {
-		if ( self::$log_enabled ) {
-			if ( empty( self::$log ) ) {
-				self::$log = new WC_Logger();
-			}
-			self::$log->add( 'komoju', $message );
-		}
-	}
-
-	/**
-	 * Check if this gateway is enabled and available in the user's country
-	 *
-	 * @return bool
-	 */
-	public function is_valid_for_use() {
-		return in_array( get_woocommerce_currency(), apply_filters( 'woocommerce_komoju_supported_currencies', array( 'JPY' ) ) );
-	}
-
-	/**
-	 * Admin Panel Options
-	 */
-	public function admin_options() {
-		if ( $this->is_valid_for_use() ) {
-			parent::admin_options();
-		} else {
-			?>
-			<div class="inline error"><p><strong><?php _e( 'Gateway Disabled', 'komoju-woocommerce' ); ?></strong>: <?php _e( 'Komoju does not support your store currency.', 'komoju-woocommerce' ); ?></p></div>
-			<?php
-		}
-	}
-
-	/**
-	 * Initialise Gateway Settings Form Fields
-	 */
-	public function init_form_fields() {
-		$this->form_fields = include( 'includes/settings-komoju.php' );
-	}
-
-	/**
-	 * Process the payment and return the result
-	 *
-	 * @param int $order_id
-	 * @return array
-	 */
-	public function process_payment( $order_id ) {
-		include_once( 'includes/class-wc-gateway-komoju-request.php' );
-		$order          = wc_get_order( $order_id );
-		$komoju_request = new WC_Gateway_Komoju_Request( $this );
-		$payment_method = sanitize_text_field($_POST['komoju-method']);
-
-		return array(
-			'result'   => 'success',
-			'redirect' => $komoju_request->get_request_url( $order, $payment_method ) 
-		);
-	}
-
-
-	/**
-	 * Payment form on checkout page
-	 */
-	public function payment_fields() {
-
-        if ( $description = $this->get_description() ) {
-            echo wpautop( wptexturize( $description ) );
+  /** @var array Array of locales */
```

Comment:
I believe default PHP indentation is 4 😄. I don't really mind either way, but it has made the diff on this PR a little harder to parse, since most of the code has been moved back two spaces and is showing as changed.

Reply:
yeah I think you are right about this one. Let me add the spaces. I actually autoformatted this to fix some misaligned stuff but ended up moving everything to 2 space indentation 🤔